ggml/src/ggml-vulkan/ggml-vulkan.cpp

```diff
@@ -2469,7 +2751,7 @@ static vk_matmul_pipeline ggml_vk_get_mul_mat_mat_pipeline(ggml_backend_vk_conte
         assert(src1_type == GGML_TYPE_F16);
         return ctx->device->pipeline_dequant_mul_mat_mat_f16[src0_type].f16acc;
```

Comment:
This coopmat2-specific code can probably be removed now? The new logic covers it.

Reply:
Nevermind, I missed that one of these has `f16` in the name and the other doesn't.

Reply:
Yeah, I'll check whether f16 helps performance with coopmat in a follow-up PR. If so it can be unified.